base/initdefs.jl

```diff
@@ -50,12 +50,20 @@ environments, repo clones, cached compiled package images, and configuration
 files. By default it includes:
 
 1. `~/.julia` where `~` is the user home as appropriate on the system;
-2. an architecture-specific shared system directory, e.g. `/usr/local/share/julia`;
-3. an architecture-independent shared system directory, e.g. `/usr/share/julia`.
-
-So `DEPOT_PATH` might be:
+2. an architecture-specific shared system directory specific to the local host,
+   e.g. `/usr/local/lib/julia`;
+3. an architecture-independent shared system directory specific to the local host,
+   e.g. `/usr/local/share/julia`;
```

Comment:
I'm actually not sure it's a good idea to look in `/usr/local` by default. I've kept `/usr/local/share/` for backward compatibility, and added `/usr/local/lib` for consistency. But this doesn't make a lot of sense as the next two dirs (4 and 5) are relative to the `julia` path so they will correspond to `/usr/local/` if Julia was installed there. And if there are two Julia copies, one in `/usr` and the other in `/usr/local`, the latter is going to interfere with the former.